net-libs/canlock/canlock-3.3.1.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="A library for creating and verifying Usenet cancel locks"
+HOMEPAGE="https://micha.freeshell.org/libcanlock/"
+SRC_URI="https://micha.freeshell.org/lib${PN}/src/lib${P}.tar.bz2"
+S="${WORKDIR}/lib${P}"
+
+LICENSE="BSD MIT"
+SLOT="0/3"
+KEYWORDS="~amd64 ~arm ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE="header-parser +legacy"
+
+BDEPEND="
+	app-alternatives/lex
+	app-alternatives/yacc
+"
+
+DOCS=( ChangeLog{,_V{0..2}} README TODO doc/sec_review.txt )
+
+QA_CONFIG_IMPL_DECL_SKIP=(
+	# Annex K functions, bug #900086
+	# optional, not implemented anywhere we care about
+	memset_s
+	explicit_memset
+)
+
+src_prepare() {
+	default
+
+	# shatest fails to build with slibtool
+	sed 's/shatest$(EXEEXT) //g' -i test/Makefile.in || die
```

Comment:
`eautoreconf` is going to clobber the `Makefile.in` modification if automake is in use.

Reply:
I'll look at better way to fixing this sed, it was there from previous ebuild and I didn't look at it in the configure fixing

Reply:
Fixed. Also, that comment was bogus: shatest fails to build because it wants static library.